odbc/src/api/data.rs

```diff
@@ -115,7 +114,23 @@ impl IntoCData for Bson {
             Bson::Int64(i) => Ok(i.to_le_bytes().to_vec()),
             Bson::Binary(b) => Ok(b.bytes),
             Bson::Decimal128(d) => Ok(d.bytes().to_vec()),
```

Comment:
As of now, we'll just send the bytes back. I think it may be better to wait for the full implementation of this type before we try to represent it concretely. Thoughts?

Reply:
Let's return the json represensation. It will make the data usable until the Rust team has Decimal128 supported "natively" in their driver